core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -883,6 +883,37 @@
   public static final String COLUMN_GROUPS = "column_groups";
   public static final String DICTIONARY_EXCLUDE = "dictionary_exclude";
   public static final String DICTIONARY_INCLUDE = "dictionary_include";
+
+  /**
+   * Table property to enable or disable local dictionary generation
+   */
+  public static final String LOCAL_DICT_ENABLE = "local_dict_enable";
+
+  /**
+   * default value for local dictionary generation
+   */
+  public static final String LOCAL_DICT_ENABLE_DEFAULT = "true";
+
+  /**
+   * Threshold value for local dictionary
+   */
+  public static final String LOCAL_DICT_THRESHOLD = "local_dict_threshold";
```

Comment:
same as above comment, provide full meaningful name. do for other properties as well.